Autopilot/PathManager/Src/landingTakeoffManager.cpp

```diff
@@ -0,0 +1,184 @@
+#include "waypointManager.hpp"
+#include "landingTakeoffManager.hpp"
+#include <math.h>
+#include "vectorClass.hpp"
```

Comment:
Don't need to include waypointManager and vectorClass as they are already included in landingTakeoffManager